ui/raidboss/data/07-dt/raid/r3s.ts

```diff
@@ -1,14 +1,153 @@
+import Conditions from '../../../../../resources/conditions';
+import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
 import { RaidbossData } from '../../../../../types/data';
 import { TriggerSet } from '../../../../../types/trigger';
 
 export type Data = RaidbossData;
 
+const headMarkerData = {
+  // Offsets: 15872,172895
+  // Vfx Path: com_share6_5s0c
+  tankBuster: '01D3',
+} as const;
+
 const triggerSet: TriggerSet<Data> = {
   id: 'AacLightHeavyweightM3Savage',
   zoneId: ZoneId.AacLightHeavyweightM3Savage,
   timelineFile: 'r3s.txt',
-  triggers: [],
+  triggers: [
+    {
+      id: 'R3S Headmarker Tankbuster',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData.tankBuster, capture: true },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'R3S Brutal Impact',
+      type: 'StartsUsing',
+      netRegex: { id: '9425', source: 'Brute Bomber', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'R3S Octuple Lariat Out',
+      type: 'StartsUsing',
+      netRegex: { id: '93D8', source: 'Brute Bomber', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Out + Spread',
+        },
+      },
+    },
+    {
+      id: 'R3S Octuple Lariat In',
+      type: 'StartsUsing',
+      netRegex: { id: '93D9', source: 'Brute Bomber', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'In + Spread',
+        },
+      },
+    },
+    {
+      id: 'R3S Octoboom Dive Proximity',
+      type: 'StartsUsing',
+      netRegex: { id: '93DE', source: 'Brute Bomber', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Away + Spread',
+        },
+      },
+    },
+    {
+      id: 'R3S Octoboom Dive Knockback',
+      type: 'StartsUsing',
+      netRegex: { id: '93DF', source: 'Brute Bomber', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Knockback + Spread',
+        },
+      },
+    },
+    {
+      id: 'R3S Murderous Mist',
+      type: 'StartsUsing',
+      netRegex: { id: '93FE', source: 'Brute Bomber', capture: false },
+      response: Responses.getBehind(),
```

Comment:
It might make sense to put a phase tracker on to control this one.  There's a `Murderous Mist` that happens during the second set of Clones/Lariats where the player is boss-tethered and actually wants to get hit by the Mist.

Reply:
My current prog is exactly one clean pull past fuse field. I'll see about a tracker of some sort for this.